pkg/ovs/ovsctl/vsctl.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ovsctl
+
+import (
+	"bufio"
+	"fmt"
+	"strings"
+
+	"k8s.io/utils/exec"
+)
+
+func ListBridges(executor exec.Interface) ([]string, error) {
+	brListOutput, err := executor.Command("ovs-vsctl", "list-br").CombinedOutput()
+	if err != nil {
+		return nil, fmt.Errorf("error when dumping ovs bridge info: %w", err)
```

Comment:
In this PR, new usages of "%w" are added while old ones are removed, why is it? Is it encouraged to use "%w" or not?

Reply:
The `%w` is only supported in `fmt`. The "%w" here caused warning.